src/main/webapp/app/shared/layouts/main/main.component.ts

```
@@ -31,6 +32,7 @@ export class JhiMainComponent implements OnInit, OnDestroy {
     isExamStarted: boolean = false;
     isTestRunExam: boolean = false;
     isCourseOverview: boolean = false;
+    isLti: boolean = false;
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)